### DISCUSSION TITLE
Installing pinned Node.js from rpm

### DISCUSSION THREAD

**Discussion by minijusl** [nodesource/distributions#1666] [Q&A] [answered]
Before repository changes it was possible to installed pinned Node.js from https://rpm.nodesource.com/pub_18.x/el/8/x86_64/nodejs-18.17.1-1nodesource.x86_64.rpm

Is it still possible for version later than 18.17.1 from new repository?

**Comment by riosje** [comment, marked as answer]
You can pin an specific version to ve installed, we have a guide in our wiki to do that.

[RPM Instructions](https://github.com/nodesource/distributions/wiki/How-to-select-the-Node.js-version-to-install#instructions-for-rpm-systems)
[DEB Instructions](https://github.com/nodesource/distributions/wiki/How-to-select-the-Node.js-version-to-install#instructions-for-debian-systems)

**Reply by minijusl** [reply]
It does not work.

```dockerfile
RUN yum install https://rpm.nodesource.com/pub_18.x/nodistro/repo/nodesource-release-nodistro-1.noarch.rpm -y
RUN yum install nodejs-18.18.0-1nodesource
```

Results in the error:
```
All matches were filtered out by modular filtering for argument: nodejs-18.18.0-1nodesource
Error: Unable to find a match: nodejs-18.18.0-1nodesource
```

**Reply by riosje** [reply]
@minijusl what distro are you using.
I've tested in fedora, and it works...
![image](https://github.com/nodesource/distributions/assets/55195249/f8f4850a-d747-4ebb-b312-559e029e70d4)

**Reply by riosje** [reply]
Ah ok, I can see now, if you're using redhat let's use this flag `--setopt=nodesource-nodejs.module_hotfixes=1` in the `yum install` 

```
 yum install nodejs-18.18.0-1nodesource --setopt=nodesource-nodejs.module_hotfixes=1 -y
```

**Reply by minijusl** [reply]
Thank you! It was the flag I have been missing.
